community/server/src/main/java/org/neo4j/server/configuration/ServerSettings.java

```diff
@@ -215,7 +215,8 @@ private ThirdPartyJaxRsPackage createThirdPartyJaxRsPackage( String packageAndMo
     Setting<Long> gc_logging_rotation_size = setting("dbms.logs.gc.rotation.size", BYTES, "20m", min(0L), max( Long.MAX_VALUE ) );
 
     @SuppressWarnings("unused") // used only in the startup scripts
-    @Description("Path of the run directory")
+    @Description( "Path of the run directory. This directory holds the pid (process id) file for when Neo4j is running in the background. "
+            + "It may be useful to mount this to the equivalent of /tmp on Linux systems, so that it is deleted when rebooting." )
```

Comment:
The _contract_ here is that `run` contains runtime state; the pidfile is the only runtime state we currently have. The bit about `/tmp` isn't strictly correct and I think that the "deleted when rebooting" is obvious to anyone who is likely to actually make use of this information.

So I suggest this instead:

> Path of the run directory. This directory holds Neo4j's runtime state, such as a pidfile when it is running in the background. It may be placed on an in-memory filesystem such as tmpfs.

Really we should have content like this on the File Locations page in the operations manual, but that page needs restructuring a bit to allow for descriptions like this.